pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/TableConfigUtils.java

```diff
@@ -363,6 +424,25 @@ public static void validateIngestionConfig(TableConfig tableConfig, @Nullable Sc
     }
   }
 
+  /**
+   * Currently only, ValueAggregators with fixed width types are allowed, so MIN, MAX, SUM, and COUNT. The reason
+   * is that only the {@link org.apache.pinot.segment.local.realtime.impl.forward.FixedByteSVMutableForwardIndex}
+   * supports random inserts and lookups. The
+   * {@link org.apache.pinot.segment.local.realtime.impl.forward.VarByteSVMutableForwardIndex only supports
+   * sequential inserts.
+   */
+  public static void validateIngestionAggregation(String name) {
+    List<AggregationFunctionType> allowed =
```

Comment:
Put this as a constant `EnumSet` (`private static final EnumSet<AggregationFunctionType> SUPPORTED_INGESTION_AGGREGATIONS`)

Reply:
done